File: camera_avfoundation/ios/camera_avfoundation/.build/arm64-apple-macosx/debug/camera_avfoundation_objc.build/DerivedSources/resource_bundle_accessor.m
#import <Foundation/Foundation.h>

NSBundle* camera_avfoundation_objc_SWIFTPM_MODULE_BUNDLE() {
    NSURL *bundleURL = [[[NSBundle mainBundle] bundleURL] URLByAppendingPathComponent:@"camera_avfoundation_camera_avfoundation_objc.bundle"];

    NSBundle *preferredBundle = [NSBundle bundleWithURL:bundleURL];
    if (preferredBundle == nil) {
      return [NSBundle bundleWithPath:@"/Users/hayotbek/Rizo/new/rizo_driver_mobile/packages/camera/camera_avfoundation/ios/camera_avfoundation/.build/arm64-apple-macosx/debug/camera_avfoundation_camera_avfoundation_objc.bundle"];
    }

    return preferredBundle;
}

File: camera_avfoundation/ios/camera_avfoundation/.build/arm64-apple-macosx/debug/camera_avfoundation.build/DerivedSources/resource_bundle_accessor.swift
import Foundation

extension Foundation.Bundle {
    static let module: Bundle = {
        let mainPath = Bundle.main.bundleURL.appendingPathComponent("camera_avfoundation_camera_avfoundation.bundle").path
        let buildPath = "/Users/hayotbek/Rizo/new/rizo_driver_mobile/packages/camera/camera_avfoundation/ios/camera_avfoundation/.build/arm64-apple-macosx/debug/camera_avfoundation_camera_avfoundation.bundle"

        let preferredBundle = Bundle(path: mainPath)

        guard let bundle = preferredBundle ?? Bundle(path: buildPath) else {
            // Users can write a function called fatalError themselves, we should be resilient against that.
            Swift.fatalError("could not load resource bundle: from \(mainPath) or \(buildPath)")
        }

        return bundle
    }()
}